_data/events.yml

```diff
@@ -215,6 +205,39 @@
   githubUrl: ""
   blogPost: ""
 
+- name: "BBQ en Boardgames Bonanza™"
+  date: 2024-08-30
+  hide: true
```

Comment:
Remove this or it will not show

**--> CREATE A NEW PR TO ADD THIS TO THE CHECKLIST SINCE IT IS THE SECOND TIME THIS HAPPENED**